### DISCUSSION TITLE
Node Red + Swag + Authelia failed login

### DISCUSSION THREAD

**Discussion by jerome6994** [authelia/authelia#5291] [Q&A] [answered]
Good evening

I have been going around in circles on the following subject for several days, even the same week.
I implemented Swag + Authelia in docker with a MYSQL database.
Everything seems to be set up correctly since the Swag service is running on all my subdomains.
When I add Authelia it also seems to be correct because almost all my subdomains work with it!
I have a problem with Node Red + Swag + Authelia.
If I put Node Red + Swag everything works but if I add Authelia then arrived at the login + mdp of Node Red it never wants to open. If I remove Authelia the login + mdp Node Red works
Have you ever had this kind of problem? If yes how to solve this major concern for me?
I'm not a computer scientist at all, just a hobby geek.
I've had the problem since I did a fresh install a few weeks ago.
If you need visual elements, don't hesitate to ask me because I don't really know what to send you
THANKS
jerome

**Comment by james-d-elliott** [comment, marked as answer]
See the latest SWAG changelog: https://github.com/linuxserver/docker-swag#versions

You may have to manually make the adjustments they made.
